src/app/contexts/ServiceContext/__mocks__/index.tsx

```diff
@@ -17,11 +16,12 @@ interface Props {
  * using service contexts.
  */
 export const ServiceContext = React.createContext({});
+
 export const ServiceContextProvider = ({
   children,
   pageLang,
```

Comment:
```suggestion
  pageLang = null,
```

I'm not sure if this is necessary? I imagine as writtent `pageLang` comes through as `undefined` instead of `null`. Probably doesn't matter though?

Reply:
I'll add it in for safety